src/impy/models/dpmjetIII.py

```diff
@@ -160,9 +158,9 @@ def _set_kinematics(self, kin):
             self._lib.dt_init(
                 -1,
                 max(kin.plab, 100.0),
```

Comment:
Also the maximal initialization energy cannot be exceeded. So, it has to be saved as well.

Reply:
dt_init can only called once, right? At least that was my impression from observing the code. MCRunRemote makes it so that _set_kinematics is called exactly once per instance.